planet/clients/orders.py

```diff
@@ -271,18 +274,67 @@ async def download_asset(self,
                              progress_bar=progress_bar)
         return dl_path
 
+    @staticmethod
+    def validate_checksum(manifest_data: dict, filenames: list, checksum: str):
```

Comment:
@sgillies had a great point but somehow with the passage of time it looks like the original comment is gone. @sgillies did you change your mind on recommending this fcn moves out of `OrdersClient` while staying in the module?

This was my response to that recommendation:
> good point @sgillies and I agree. 
> 
> @mkshah605 if it feels like an easy change, go for it. If you think it will take you a while (>1 hour) to implement, make a ticket for moving the function out of the client, link to that ticket here, and feel free to continue with the PR.

Reply:
@jreiberkyle I can't seem to find this comment anymore either, but @sgillies  and I talked about it separately- I wasn't really sure what it would involve/if there were other considerations we needed to take into account as a group given our current timeline, but I agree it's a great idea and am happy to implement. Sean made a [new ticket about it here](https://github.com/planetlabs/planet-client-python/issues/521). 

Could (either of you) clarify on what you mean by "out of the Orders Client into the current module"? Where/what file would this function ideally sit in so that it could be more generally available to users?

Reply:
The function `validate_checksum` would still be in `orders.py` but would not be a method of the `OrdersClient` class.

Reply:
Right, I moved my comment to #521 and deleted it here so it wouldn't be a distraction.

Reply:
awesome!